src/kfs_lib/kfs_chmod.c

```diff
@@ -1,10 +1,11 @@
 #include "kfs_chmod.h"
 
 int kfs_chmod(const char *path, mode_t mode) {
+    log_info("kfs_chmod start");
     int res = 0;
     char fpath[PATH_MAX];
     fullpath(fpath, path);
-    log_debugf("kfs_chmod path : %s\n", fpath);
+    log_debugf("    kfs_chmod path : %s\n", fpath);
```

Comment:
is it possible to make this space to be a macro? to do now put always fixed set of spaces but "SHIFT kfs_cmod path"?